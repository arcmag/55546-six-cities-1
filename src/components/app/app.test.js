import React from 'react';
import renderer from 'react-test-renderer';

import App from './app.jsx';

describe(`Test App`, () => {
  it(`Test App renderer`, () => {
    const tree = renderer.create(<App rentList={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
