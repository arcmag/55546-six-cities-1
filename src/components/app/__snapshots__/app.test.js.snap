// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Test App renderer 1`] = `
<div>
  <h1>
    React application
  </h1>
</div>
`;
